core/metrics/src/main/scala/org/typelevel/otel4s/metrics/ObservableInstrumentBuilder.scala

```diff
@@ -61,4 +63,19 @@ trait ObservableInstrumentBuilder[F[_], A, Instrument] {
   def createWithCallback(
       cb: ObservableMeasurement[F, A] => F[Unit]
   ): Resource[F, Instrument]
+
+  /** Creates an asynchronous instrument that an effect that produces a number
+    * of measurements
+    *
+    * @param measurements
+    *   Effect that produces a number of measurements
+    */
+  def create(measurements: F[List[Measurement[A]]])(implicit
+      F: Monad[F]
```

Comment:
The `F: Monad[F]` is available in the `Observable*BuilderImpl` as an implicit parameter of the case class: `F: Async[F]`. 

We can remove the default implementation and make one in the `Observable*BuilderImpl` case class similar to `def createWithCallback`. This would help eliminate the need for an implicit `Monad[F]` at the call sites.

```scala
def create(measurements: F[List[Measurement[A]]]): Resource[F, Instrument]
```

Reply:
The java classes are not very friendly to generic programming.